src/Bundle/Tests/Functional/Maker/MakeGridTest.php

```diff
@@ -43,6 +46,19 @@ public function it_can_create_grids(): void
         $this->assertSame(self::getPriceGridExpectedContent(), \file_get_contents(self::tempFile(self::PRICE_GRID_PATH)));
     }
 
+    /** @test */
+    public function it_can_create_grids_with_a_simple_resource(): void
```

Comment:
I think we don't have "simple resource" term, which will referrer to doctrine-less resource.

How about,
```patch
---    public function it_can_create_grids_with_a_doctrine_entity(): void
---    public function it_can_create_grids_with_a_simple_resource(): void
+++    public function it_can_create_grids: void
+++    public function it_can_create_grids_without_doctrine(): void
```

Reply:
Changed it :)